critiquebrainz/data/model/user_test.py

```diff
@@ -148,21 +148,21 @@ def test_reviews_property(self):
         self.assertEqual(len(user.reviews), 0)
 
     def test_avatar_property(self):
-        user = User(display_name=u'Tester', email=u"example@example.org")
+        user = User(display_name=u'Tester', email=u"example@example.org", id=u"ea1c2bf3-f246-d47c-f6a4-e5f6e7b8d9e0")
         db.session.add(user)
         db.session.commit()
 
         # By default show_gravatar attribute should be set to False
         self.assertFalse(user.show_gravatar)
         # so avatar property returns generic avatar
-        self.assertEqual(user.avatar, "https://gravatar.com/avatar/placeholder?d=mm")
+        self.assertEqual(user.avatar, "https://gravatar.com/avatar/476c357ea2c081b90bf2a6659d0fbd26?d=identicon")
```

Comment:
Rather than hard-coding MD5 hash here I'd rather see it calculated based on the user ID.